### ISSUE TITLE
Can't private message on Chatterino

### ISSUE THREAD

**Issue by victorleporc** [Chatterino/chatterino2#2132]
I just can't send messages even with all the settings disabled. It says "Your settings prevent you from sending this whisper." 
I disabled all privacy settings and it's not working. See there: https://prnt.sc/v6jrvu (written in french but you can match it with your own settings) https://prnt.sc/v6jvyx

*Chatterino version is 2.2.3-beta but even with the older versions I wasn't able to send private messages through Chatterino
Running on Windows 10

By the way I'm a Chatterino supporter ;) (@victorleporc_)

**Comment by TranRed** [comment]
duplicate of #1526 ?!

**Comment by Mm2PL** [comment]
This was blocked by twitch, not Chatterino. Your best bet is to ask twitch support :(

**Comment by alazymeme** [comment]
If you cannot send whispers from your Twitch account and you've verified that the relevant settings are disabled, it may be due to your account being too new to be able to whisper strangers. If you wish to whisper a user, you must first add them as a friend. This was a spam protection measure put in place by Twitch. This is not a Chatterino bug.

**Comment by victorleporc** [comment]
I really don't know what to do, asking twitch support about a 3rd party app isn't strange to do?

The thing is that my account is 1.5 years old, can send whispers to strangers, all settings are disabled and i'm trying to whisper a user already added as a friend.

**Comment by leon-richardt** [comment]
> I really don't know what to do, asking twitch support about a 3rd party app isn't strange to do?

In a sense, it's the "next best thing" to do after asking here. For sending whispers, Chatterino is dependent on the interface Twitch provides. What happens behind that interface is completely in Twitch's control.

**Comment by alazymeme** [comment]
Funnily enough you may be able to send whispers on the twitch website. It seems to be a specific IRC limit that's been implemented (not 100% sure on this).

I believe it would have something to do with the spambots that were whispering users a few months ago (that would've been connecting via IRC).

**Comment by daiexd** [comment]
> Funnily enough you may be able to send whispers on the twitch website.

Messages seem to rarely be blocked when sending through the twitch website, so the next best thing you can do is just have a /whispers tab and open in browser whenever you need to send a whisper. Not as efficient but unfortunately the next best option.
